### ISSUE TITLE
Logistic vs cross-entropy

### ISSUE THREAD

**Issue by amueller** [afshinea/stanford-cs-229-machine-learning#11]
I don't think it makes sense to plot different functions for logistic loss and cross-entropy loss as they are essentially two names for the same thing. Sometimes these names are used to differentiate between an overparametrized (softmax) version vs a non-overparametrized version but that's independent of the loss used. In particular the two formulas that are shown are equivalent (one assumed y is +1, -1 the other assumes y is 0, 1). Showing different graphs for the same formula seems confusing.
